sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsDiskStore.scala

```diff
@@ -38,6 +39,13 @@ class RapidsDiskStore(diskBlockManager: RapidsDiskBlockManager)
     extends RapidsBufferStoreWithoutSpill(StorageTier.DISK) {
   private[this] val sharedBufferFiles = new ConcurrentHashMap[RapidsBufferId, File]
 
+  private def reportDiskAllocMetrics(metrics: GpuTaskMetrics): Unit = {
```

Comment:
nit: This is only used for debugging right? Do we want to keep this around in the final version? If so, then I would propose that we make this return a string so that the logDebug can decide if it even gets called or not.